### PR TITLE
feat: add the clusters and clusternodes validator

### PR DESCRIPTION
**Pull request by wangyizhi1**
#### What type of PR is this?

/kind feature

#### What does this PR do?

feat: add the clusters and clusternodes validator

#### Which issue(s) does this PR fix?

Fixes part of #302 
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/clusterlink/network-manager/controller.go [hunk 1]
```diff
@@ -123,7 +123,19 @@ func (c *Controller) Reconcile(ctx context.Context, request reconcile.Request) (
 	}
 
 	str := c.NetworkManager.GetConfigsString()
-	klog.V(4).Infof(str)
+	klog.V(6).Infof(str)
```
Comment (hanweisen) [inline pkg/clusterlink/network-manager/controller.go]: add more message in Infof

Comment (wangyizhi1) [reply pkg/clusterlink/network-manager/controller.go]: Done, please review again.

**Pull request merged**